apps/preview/next/pages/showcases/RatingForms/ProductRating.tsx

```diff
@@ -0,0 +1,66 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useState } from 'react';
+import { SfModal, SfButton, SfIconClose, useDisclosure, SfRatingButton } from '@storefront-ui/react';
+
+export default function ProductRating() {
+  const { isOpen, open, close } = useDisclosure({ initialValue: true });
+  const [rating, setRating] = useState(0);
+
+  return (
+    <>
+      <SfButton type="button" className="absolute right-1/2 top-1/2 translate-x-[50%]" onClick={open}>
+        Open rating modal again
+      </SfButton>
+      <SfModal
+        open={isOpen}
+        onClose={close}
+        className="min-w-[376px] md:w-[480px]"
+        as="section"
+        role="alertdialog"
+        aria-labelledby="modalTitle"
+        aria-describedby="modalDesc"
+      >
+        <header>
+          <SfButton
+            square
+            variant="tertiary"
+            className="!text-neutral-500 absolute right-2 top-2"
+            aria-label="Close product rating modal"
+            onClick={close}
+          >
+            <SfIconClose />
+          </SfButton>
+          <h3 id="modalTitle" className="font-bold typography-headline-4 md:typography-headline-3 mb-8">
```

Comment:
Use `useId` to generate ids, please. Same for `modalDesc` id.

```
const modalTitle = useId()
const modalDesc = useId()
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2849/commits/d3797690dbd108bf5bf4029e2c4270327ee10e2a